cli/flox-watchdog/src/main.rs

```diff
@@ -176,12 +172,13 @@ fn run(args: Cli) -> Result<(), Error> {
         spawn_gc_logs(log_dir);
     }
 
-    // Listen for a notification, getting an error if we should terminate
+    debug!("waiting for termination");
```

Comment:
Nit: waiting for termination of what exactly?

Reply:
@zmitchell if you want to push a tweak I'll merge it